jenkins/swift-functional-tests/20-install-openstack.sh

```diff
@@ -9,20 +10,37 @@ enable_service key mysql s-proxy s-object s-container s-account
 SCREEN_LOGDIR="\${DEST}/logs"
 EOF
     cp jenkins/${JOB_NAME%%/*}/extras.d/55-swift-sproxyd.sh devstack/extras.d/55-swift-sproxyd.sh
+    if [[ $DEVSTACK_BRANCH == "stable/icehouse" ]]; then
+        # Workaound depencies version conflict :
+        # last python-glanceclient (0.17 at the time of writing), wich gets installed by default,
+        # requires keystoneclient >= 1.0.0
+        # whereas keystone requires keystoneclient <= 0.11.2
+        # python-glanceclient >= 0.13.1 is required by openstackclient 0.4.1
+        sudo pip install python-glanceclient==0.13.1
+    fi
     ./devstack/stack.sh
 }
 
+function ubuntu_common {
+    wget https://bootstrap.pypa.io/ez_setup.py -O - | sudo python;
```

Comment:
I'd copy this file in the repository (somewhere along this script) instead. No need to rely on external sources too much.